### PR TITLE
String parsing

### PR DESCRIPTION
**Pull request by TrySound**
Added css-list
Parses all separators (' ', '\n', '\t', ',', '/')

### CODE REVIEW

**Comment by ben-eb** [comment]
Thanks, @TrySound. I'll take a look at this tonight. :smile:

Did you run JSHint? I noticed a missed semicolon.

File: lib/convert.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}, {
+    // Time
+    's': 1000,
+    'ms': 1
+}];
+
+function dropLeadingZero (number) {
+    var value = number.toString();
+
+    if(value[0] === '0' && number % 1 !== 0) {
```
Comment (ben-eb) [inline lib/convert.js]:
```js
if (value[0] === '0' && number % 1)
```

File: lib/eachValue.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+var balanced = require('balanced-match');
+var list = require('css-list');
+
+module.exports = function eachValue (value, callback) {
+    return list.map(value, [' ', '\n', '\t', ',', '/'], function(value, type) {
+        var name,
+            match,
+            index;
+
+        if(type === null) {
```
Comment (ben-eb) [inline lib/eachValue.js]:
```js
if (type === null) {

File: lib/eachValue.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            return callback(value);
+        }
+
+        if(type === 'func') {
```
Comment (ben-eb) [inline lib/eachValue.js]:
```js
if (type === 'func') {
```

File: lib/eachValue.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            index = value.indexOf('(');
+            name = value.substring(0, index);
+            if(name.indexOf('calc') > -1) {
```
Comment (ben-eb) [inline lib/eachValue.js]:
```js
if (~name.indexOf('calc')) {
```

File: lib/eachValue.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+                match = balanced('(', ')', value);
+                if(match) {
```
Comment (ben-eb) [inline lib/eachValue.js]:
```js
if (match) {
```

File: lib/parseUnit.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+// From longer to shorter!
+var units = ['rem', 'em', 'ex', 'in', 'cm', 'mm', 'pt', 'pc', 'px', 'ms', 's', '%'];
+
+module.exports = function (value) {
+    var max, index;
+
+    for(max = 3; max !== 0; max -= 1) {
```
Comment (ben-eb) [inline lib/parseUnit.js]:
```js
for (max = 3; max !== 0; max -= 1) {
```

File: lib/parseUnit.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+        index = units.indexOf(value.slice(-max));
+        if(index !== -1) {
```
Comment (ben-eb) [inline lib/parseUnit.js]:
```js
if (index !== -1) {
```

**Comment by ben-eb** [comment]
Other than the comments, LGTM. :+1:

File: lib/convert.js (hunk 1 above)
Comment (TrySound) [inline lib/convert.js]: I see code, but what's wrong? :)

Comment (TrySound) [inline lib/convert.js]: I will add jscs

**Pull request merged**